README.md

```diff
@@ -12,38 +12,92 @@ The `amazon_cloud_code_generator` relies on the CloudFormation client. Hence, th
+
+- `data/` - Contains the content that will be copied over the generated collection. A high level overview of the subdirectory's content is shown below.
+    ```
+    ├── changelogs  # Changelog fragments that will be used for the new collection release
+    │   ├── plugins
+    │   │   └── module_utils # Modules utils that are used by the collection plugins
+    │   │       ├── core.py
+    │   │       ├── utils.py
+    │   │       └── waiters.py
+    │   └── tests  # Contains the units and integrations tests
+    │       ├── config.yml
+    │       ├── integration
+    │       │   ├── constraints.txt
+    │       │   ├── requirements.txt
+    │       │   └── targets
+    │       │       ├── logs
+    │       │       │   ├── aliases
+    │       │       │   └── tasks
+    │       │       │       └── main.yml
+    │       │       ├── ...
+    │       └── unit
+    │           ├── constraints.txt
+    │           ├── module_utils
+    │           │   ├── test_core.py
+    │           │   └── test_utils.py
+    │           └── requirements.txt
+    ```
+
+- `templates/` - Hosts the templates files that will be used to generate each collection's plugin. 
+    ```
+    ├── default_module.j2
+    └── header.j2
+    ```
+
+The modules generated by the `amazon_cloud_code_generator/` will be made availabale in the `cloud` subdirectory by default. You can use the following command to do so:
+
+```python -m amazon_cloud_code_generator.cmd.refresh_modules```
+
+A different target directory can be specified by using:
+
+```python -m amazon_cloud_code_generator.cmd.refresh_modules --target-dir /somewhere/else```
 
 When available, the documented EXAMPLE block of the generated modules can be updated by using the following command:
 
-```tox -e refresh_examples --target-dir /somewhere/else```
-
-This information is scraped from the tests/ directory.
+```python -m amazon_cloud_code_generator.cmd.refresh_examples```
 
 ## How to refresh the amazon.cloud collection
 
-Fork the original [``amazon.cloud collection``](https://github.com/ansible-collections/amazon.cloud) repository to your account on GitHub. Clone the repository from your fork:
+Fork the original [``amazon.cloud collection``](https://github.com/ansible-collections/amazon.cloud) repository to your account on GitHub. Ensure to have the path `~/.ansible/collections/ansible_collections/amazon/cloud` and clone the repository from your fork:
 ```
 mkdir -p ~/.ansible/collections/ansible_collections/amazon/cloud
 git clone https://github.com/YOUR_USER/amazon.cloud ~/.ansible/collections/ansible_collections/amazon/cloud
 ```
 
-Refresh the content of the modules moving in the repository path:
+Refresh the content of the modules moving in the repository path using `tox -e refresh_modules`.
```

Comment:
It's unclear to me what `moving in the repository path` means here.  Would that be something like, "Refresh the content of the modules by running `tox -e refresh_modules` from the root of the repository"?

Reply:
Rephrased like this: ``Refresh the content of the modules by moving in the repository path `~/.ansible/collections/ansible_collections/amazon/cloud` and running `tox -e refresh_modules`.
``